app/models/manageiq/providers/openstack/network_manager/cloud_network.rb

```diff
@@ -10,6 +10,9 @@ def self.remapping(options)
   end
 
   def self.raw_create_network(ext_management_system, options)
+    # TODO: remove this log line once this uses the task queue, as the task queue has its own logging
+    _log.info "Command: ManageIQ::Providers::Openstack::NetworkManager::" \
+              "CloudNetwork.raw_create_network, Args: #{options.inspect}"
```

Comment:
Thanks for the review!  I've chosen the first option, since the intent is to change this functionality to use task_queues in the errata release.  Tasks queues generate log messages like these automatically, so I figure it's better to not create a log_prefix method that won't be needed in the long run and is more likely to be left behind as clutter.